### PR TITLE
[MR-855] Prune unused api endpoints, rename db types, lots of general renaming

### PR DESCRIPTION
**Pull request by macrael**
## Summary
This continues the refactor work, getting rid of GraphQL operations that we aren’t using anymore and clening up the database as well.

At this point, we just need to clean up the FE once Maz’s work is in and we’re in good shap. There’s a little more organizational cleanup, but the bulk of the work is done. 

We probably should rename the proto stuff, too.

### CODE REVIEW

File: services/app-api/resolvers/indexHealthPlanPackages.ts [hunk 1]
```diff
@@ -21,26 +21,24 @@ export function indexHealthPlanPackagesResolver(
...
         if (isStoreError(results)) {
-            const errMessage = `Issue finding a draft submission of type ${results.code}. Message: ${results.message}`
+            const errMessage = `Issue finding packages of type ${results.code}. Message: ${results.message}`
             logError('indexHealthPlanPackages', errMessage)
             setErrorAttributesOnActiveSpan(errMessage, span)
             throw new Error(errMessage)
         }
 
-        const submissions: HealthPlanPackageType[] = results
+        const pacakges: HealthPlanPackageType[] = results
```
Comment (haworku) [inline services/app-api/resolvers/indexHealthPlanPackages.ts:39]: minor typo

Comment (macrael) [reply services/app-api/resolvers/indexHealthPlanPackages.ts:39]: I’m shocked there aren’t more of those

**Comment by macrael** [comment]
The tests are rightly failing, my hack to try and nix the old updates API isn’t actually working. I think that since maz’s work is close I’ll just wait for his work to land then put my changes on top of his.

File: services/app-api/resolvers/indexHealthPlanPackages.ts (hunk 1 above)
Comment (haworku) [reply services/app-api/resolvers/indexHealthPlanPackages.ts:39]: Oh there is one that's been bugging me with `initiallySubmittedAt` being spelled wrong . if you feel like fixing typos :trollface: 

Good aside, all of this work looks good I do think it makes things clearer.

**Review by haworku — APPROVED** [review]
@macrael feel free to ping me for rereview if you end up needing it

File: tests/cypress/support/stateSubmissionFormCommands.ts [hunk 2]
```diff
@@ -258,12 +256,6 @@ Cypress.Commands.add(
                 'exist'
             ) 
         } else if (buttonKey === 'CONTINUE'){
-            if (waitForLoad){
-                cy.wait('@updateDraftSubmissionMutation')
-                cy.wait('@fetchHealthPlanPackageQuery')
-            }
```
Comment (haworku) [inline tests/cypress/support/stateSubmissionFormCommands.ts:263]: just FYI this was only added to address the flakes that happened after OTEL. Let's keep an eye on Cypfess

**Pull request merged**